include/mbedtls/ctr_drbg.h

```diff
@@ -8,8 +8,11 @@
  * Recommendation for Random Number Generation Using Deterministic Random
  * Bit Generators</em>.
  *
- * The Mbed TLS implementation of CTR_DRBG uses AES-256 as the underlying
- * block cipher.
+ * The Mbed TLS implementation of CTR_DRBG uses AES-256 (default) or AES-128
+ * as the underlying block cipher.
+ *
+ *  \warning values generated by AES-128 based CTR_DRBG may be less secure
+ *           as they are easier to guess.
```

Comment:
This warning is not informative. I think we want something like this — @yanesca please comment and improve.

> Using AES-128 for the random number generator limits the security of all generated keys and all operations that use random values to 128-bit security.

Do we need to invoke the birthday paradox as well?

Reply:
I'm more than happy to adopt your version. I'll wait to see inputs from @yanesca  before updating to avoid multiple updates

Reply:
I am happy with the wording. Birthday attacks don't apply here, but even if they would, the reseed limit would prevent them (our reseed limit is even more conservative than the one prescribed by NIST).

Reply:
Thanks @yanesca  and @gilles-peskine-arm , changed to text suggested by Gilles.